src/firefly/js/visualize/ui/GatorProtocolRootPanel.jsx

```diff
@@ -0,0 +1,202 @@
+            </Typography>
+        );
+    }
+    else if (error)  {
+        return (
+            <Stack>
+                <Typography level='h3' color='warning' sx={{textAlign:'center', mt:5}}>Error</Typography>
+                <Typography color='warning' sx={{textAlign:'center', mt:5}}>{error}</Typography>
+            </Stack>
+        );
+    }
+    return <div/>;
+}
+
+function Services({serviceUrl, servicesShowing, gpOps, onGatorProtoServiceOptionSelect,title} ) {
+    const [extraStyle,setExtraStyle] = useState({overflow:'hidden'});
+    const enterUrl=  useFieldGroupValue('enterUrl')[0]();
+
+    useEffect(() => {
+        if (servicesShowing) {
+            setTimeout(() => setExtraStyle({overflow:'visible'}), 250);
+        }
+        else {
+            setExtraStyle({overflow:'hidden'});
+        }
+    }, [servicesShowing]);
+
+    return (
+        <Stack sx={{
+            height: servicesShowing ? 'auto' : 0,
+            pb: servicesShowing ? 1.5 : 0,
+            justifyContent:'space-between',
+            alignItems:'center',
+            transition: 'all .2s ease-in-out', //to animate height changes (hide/show Services)
+            ...extraStyle}}>
+            <Stack direction='row' spacing={1} sx={{alignItems:'center', width:1}}>
+                <Stack alignItems='flex-start' spacing={1}>
+                    <Typography {...{level:'title-lg', color:'primary', sx:{width:'17rem', mr:1} }}>
+                        Select Source
+                    </Typography>
+                    <SwitchInputField {...{ size:'sm', endDecorator:'Enter my URL', fieldKey:'enterUrl',
+                        initState:{value:false},
+                        sx: {
+                            alignSelf: 'flex-start',
+                            '--Switch-trackWidth': '20px',
+                            '--Switch-trackHeight': '12px',
+                        },
+                    }} />
+                </Stack>
+                <Stack>
+                    {enterUrl ? (
+                        <InputField orientation='horizontal'
+                                    placeholder='Enter SIAv2 Url'
+                                    value={serviceUrl}
+                                    actOn={['enter']}
+                                    tooltip='enter SIAv2 URL'
+                                    slotProps={{input:{sx:{width:'40rem', height: '2.25rem'}}}}
+                                    onChange={(val) => {
+                                        onGatorProtoServiceOptionSelect(val);
+                                        addGatorProtoUserService(val.value);
+                                    }}
+                        />
+
+                    ) : (
+                        <ListBoxInputFieldView {...{
+                            sx:{'& .MuiSelect-root':{width:'46.5rem'}},
+                            options:gpOps, value:serviceUrl,
+                            placeholder:'Choose SIAv2 Service...',
```

Comment:
"SIAv2" doesn't seem to belong here either.  I'm not sure that this placeholder is actually displaying anywhere, though - maybe it's overridden elsewhere in the code?